client/src/components/map.module.scss

```diff
@@ -2,4 +2,5 @@
   height: 676px;
   margin-bottom: 29px;
   max-width: revert;
+  margin-top: 50px;
```

Comment:
This seems pretty specific so I hope the parent container is positioned relative. Might want to double check responsiveness later. Not blocking for this PR to be merged tho.

Reply:
True - this can and should be revisited; for now the purpose is to get this in a popup; current specs don't have a popup at all actually so this will be stripped out